docs/REQUIREMENTS.md

```
@@ -59,6 +59,32 @@ Checks PHP CLI configuration values against expected minimums:
 
 Checks for the availability of the `mariadb` or `mysql` client and validates the version against client-specific minimums (MariaDB: >= 10.4.3, MySQL: >= 8.0.17).
 
+### Database grants
+
+Validates database user permissions based on the configured `database_manager_type`. This check requires the feature deployment autoload to be loaded.
+
+**Root mode** (`default` / `root`): Connects to the database server and verifies that all required grants are set on global level (`*.*`):
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)